src/components/settings/SettingsLMSTab/utils.js

```diff
@@ -2,8 +2,9 @@ import { logError } from '@edx/frontend-platform/logging';
 
 export function buttonBool(config) {
   let returnVal = true;
-  Object.values(config).forEach(value => {
-    if (!value) {
+  Object.entries(config).forEach(entry => {
+    const [key, value] = entry;
+    if ((key !== 'displayName' && key !== 'degreedFetchUrl') && !value) {
```

Comment:
since this is a generic utils file, do we want a degreed specific handling? I would maybe add a comment as to why that's included or move/make a function specific for the degreed config component